README.md

```diff
@@ -1,3 +1,7 @@
+:warning: **The scripts in this repository are no longer needed as of [User Activity Monitoring plugin version 1.3](https://docs.cloudbees.com/docs/release-notes/latest/plugins/user-activity-monitoring-plugin/1.3).**
```

Comment:
Version number noted here (2×) does not match that in the PR branch; which is right?

Reply:
1.4 is the release that went out yesterday, but 1.3 is the version with the corresponding changes, so the release notes for 1.3 have the relevant message. When I initially filed the PR I forgot about the intermediate 1.3 release.